armi/reactor/components/__init__.py

```diff
@@ -151,20 +151,28 @@ def __init__(
         )
         self._linkAndStoreDimensions(components, modArea=modArea)
 
-    def getComponentArea(self, cold=False):
+    def getComponentArea(self, cold=False, Tc=None):
         """
         Get the area of this component in cm^2.
 
         Parameters
         ----------
         cold : bool, optional
             If True, compute the area with as-input dimensions, instead of thermally-expanded.
+        Tc : float, optional
+            Temperature in C to compute the area at
         """
+        if cold and Tc is not None:
```

Comment:
```suggestion
        if cold is not False and Tc is not None:
```

Reply:
This is a VERY minor point.  And we don't have to do this.

I'm just thinking if the cold temp is 0 degrees C, then `if cold` will evaluate to `False`, instead of `True`.  Does that make sense?

Reply:
`cold` is a boolean not a temperature so I'm not sure I follow the comment?